src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -242,6 +242,49 @@ def _parameters(registry_name,
     return parameters
 
 
+def get_credentials(cmd, auth_mode=None, credentials=None):
```

Comment:
Move this method to the end, close to `get_validate_platform`. Also this is a generic util file, use a more descriptive naming.

You may also create a new file called `_task_utils` so you don't need to group task specific utils with these more generic ones.